spring-aot/src/main/java/org/springframework/cloud/function/FunctionTypeProcessor.java

```diff
@@ -60,6 +61,17 @@ void process(ConfigurableListableBeanFactory beanFactory, NativeConfigurationReg
 											TypeAccess.DECLARED_METHODS, TypeAccess.PUBLIC_METHODS);
 						}
 					});
+			registry.reflection().forType(RootBeanDefinition.class).withAccess(TypeAccess.DECLARED_FIELDS);
```

Comment:
Could we just add the required field instead of all of them?
Also if that's static, maybe that should go to `FunctionHints` instead of here.